rosdep/python.yaml

```
@@ -5003,6 +5003,9 @@ python3-termcolor:
   fedora: [python3-termcolor]
   gentoo: [dev-python/termcolor]
   ubuntu: [python3-termcolor]
+python3-tk:
+  debian: [python3-tk]
+  ubuntu: [python3-tk]
```

I have no idea what I was thinking. I wish I would have explained the reasoning there a little more.

For reference, here is the PR: https://github.com/ros/rosdistro/pull/3325

`python3-tk` seems to add the `tkinter` module on Ubuntu. Lets just do the same thing on Fedora.

```suggestion
  fedora: [python3-tkinter]
  ubuntu: [python3-tk]
```